mlflow/pipelines/utils/step.py

```diff
@@ -76,9 +76,15 @@ def display_html(html_data: str = None, html_file_path: str = None) -> None:
         import shutil
         import subprocess
 
-        if os.path.exists(html_file_path) and shutil.which("open") is not None:
+        # Use xdg-open in Linux environment
+        if shutil.which("xdg-open") is not None:
+            open_tool = "xdg-open"
+        elif shutil.which("open") is not None:
+            open_tool = "open"
```

Comment:
```suggestion
        open_tool = shutil.which("open") or shutil.which("xdg-open")
```

Reply:
I updated the code to assign the result of which to `open_tool` directly in the if block instead of a separate line.